usr/src/uts/common/fs/zfs/metaslab.c

```diff
@@ -199,6 +200,43 @@ static void metaslab_set_fragmentation(metaslab_t *);
 kmem_cache_t *metaslab_alloc_trace_cache;
 
 /*
+ * How many TXG's worth of updates should be aggregated per TRIM/UNMAP
+ * issued to the underlying vdev. We keep two range trees of extents
+ * (called "trim sets") to be trimmed per metaslab, the `current' and
```

Comment:
Sorry about that, old habit from writing tons of TeX markup. I can replace these if you insist.